src/components/SignUp.js

```diff
@@ -75,15 +76,15 @@ class Register extends React.Component {
       this.setState({
         data: {
           ...this.state.data,
-          name: this.state.data.name === null ? '' : this.state.data.name,
+          name: !this.state.data.name ? '' : this.state.data.name,
```

Comment:
lets not use not_condition as much as possible... it makes code readablity a bit difficult